### PR TITLE
Update setting the values step

### PR DESCRIPTION
**Pull request by subodh101**
What?
- Remove the make `set-values` rule step as it creates confusion of writing the same source of truth at multiple places.

Why?
- more user friendly

Additionally, fix the list step numbers.

### CODE REVIEW

**Comment by subodh101** [comment]
[NOTE] waiting on https://github.com/kubeflow/gcp-blueprints/pull/120

File: content/en/docs/gke/deploy/management-setup.md [hunk 1]
```diff
@@ -49,26 +49,12 @@ to manage GCP infrastructure using GitOps.
...
-    kpt cfg set ./instance location LOCATION
-    kpt cfg set ./instance gcloud.core.project PROJECT
-  
-    kpt cfg set ./upstream/management name NAME
-    kpt cfg set ./upstream/management location LOCATION
-    kpt cfg set ./upstream/management gcloud.core.project PROJECT
-
-   ```
-
-   * Where **NAME**, **LOCATION**, **PROJECT** should be the actual values for your deployment
```
Comment (Bobgy) [inline content/en/docs/gke/deploy/management-setup.md:66]: Would you mind keeping this line somewhere?

**Comment by Bobgy** [comment]
/hold
wait for kubeflow/gcp-blueprints#120 first

**Comment by jlewi** [comment]
I think we need to wait for a KF 1.1. branch to be cut before merging this PR as these instructions will only apply to releases after 1.1. So we need to wait until master is the docs for 1.2.

**Comment by Bobgy** [comment]
Shall we close this too?

**Pull request closed without merging**